components/aws/sagemaker/common/_utils.py

```diff
@@ -228,6 +228,14 @@ def get_image_from_job(client, job_name):
 
 
 def create_model(client, args):
+    request = create_model_request(args)
+    create_model_response = client.create_model(**request)
```

Comment:
Please put it inside `try/catch` and handle the exception accordingly.

Reply:
check `create_training_job` for reference.

Reply:
done